src/machine/board_feather-stm32f405.go

```diff
@@ -113,7 +118,30 @@ const (
 	UART_TX_PIN = UART0_TX_PIN //
 )
 
-func initUART() {}
+var (
+	UART1 = UART{
+		Buffer:          NewRingBuffer(),
+		Bus:             stm32.USART3,
+		AltFuncSelector: stm32.AF7_USART1_2_3,
+	}
+	UART2 = UART{
+		Buffer:          NewRingBuffer(),
+		Bus:             stm32.USART6,
+		AltFuncSelector: stm32.AF8_USART4_5_6,
+	}
+	UART3 = UART{
+		Buffer:          NewRingBuffer(),
+		Bus:             stm32.USART1,
+		AltFuncSelector: stm32.AF7_USART1_2_3,
+	}
+	UART0 = UART1
+)
+
+func initUART() {
+	UART1.Interrupt = interrupt.New(stm32.IRQ_USART3, UART1.handleInterrupt)
+	UART2.Interrupt = interrupt.New(stm32.IRQ_USART6, UART2.handleInterrupt)
+	UART3.Interrupt = interrupt.New(stm32.IRQ_USART1, UART3.handleInterrupt)
```

Comment:
Should we be automatically installing these interrupts for the non-default UARTs (`UART2`/`UART3`)? The default `UART1` is the only one required by other packages (e.g. `runtime.putchar()`)

Reply:
> Should we be automatically installing these interrupts for the non-default UARTs 

Possibly not? If a developer wants to use them for a specific application, they can do that themselves.

Reply:
Probably yes. This is a bit of magic code that gets optimized away if the UART is not in use (so no interrupt should be registered if you don't use UART2 for example).